### PR TITLE
Add service models and modify documentation

### PR DESCRIPTION
**Pull request by KiranSatyaRaj**
adding primary service models to the documentation and modifying it for a better understanding and readability.

Signed-off-by: Kiran Satya Raj <90622705+KiranSatyaRaj@users.noreply.github.com>

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

File: content/en/cloud-computing.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ tags: ["infrastructure", "", ""]
 Cloud computing is a model that offers compute resources like CPU, network, and disk capabilities on-demand over the internet. 
 Cloud computing gives users the ability to access and use computing power in a remote physical location. 
 Cloud providers like AWS, GCP, Azure, DigitalOcean, and others all offer third parties 
-the ability to rent access to compute resources in multiple geographic locations. 
+the ability to rent access to compute resources in multiple geographic locations.
+With a layer of [abstraction](/abstraction/) built on top of one another, this architecture is the foundation
```
Comment (iamNoah1) [inline content/en/cloud-computing.md:14]: We are not speaking of Cloud Computing as an architecture, hence introducing architecture here might confuse people. Rather stick to model. Besides that, while what you add here might be true and I get what you mean, I don't believe that it really adds value to the definition of Cloud Computing. I guess that people who are not familiar with everything might be rather confused how abstractions built on top of one another can be a foundation for IaaS. PaaS and SaaS. TBH, I would not add the last sentence.

File: content/en/cloud-computing.md [hunk 2]
```diff
@@ -23,6 +25,5 @@ Cloud computing allows organizations to outsource some portion of their computin
 
 Cloud providers offer organizations the ability to rent compute resources on-demand and pay for usage. 
 This allows for two major innovations: 
-organizations can try things without wasting time planning and spending money or resources on new physical infrastructure 
-and they can [scale](/scalability/) as needed and on-demand. 
-Cloud computing allows organizations to adopt as much or as little infrastructure as they need. 
+organizations can focus on their product or service without a needless wait, planning, and spending money or resources on new physical infrastructure. They can [scale](/scalability/) as needed and on-demand.
```
Comment (iamNoah1) [inline content/en/cloud-computing.md:28]: nice :)

**Review by iamNoah1 — CHANGES_REQUESTED** [review]
Thanks @KiranSatyaRaj for your contribution. I have some change requests :)

File: content/en/cloud-computing.md (hunk 1 above)
Comment (CathPag) [reply content/en/cloud-computing.md:14]: +1 on keeping it simple. If we don't need it, let's drop it. We are trying to create some sort of minimal viable definition. What do people need to know to understand the concept? Once they get that, they can dig deeper with other resources.

**Review by KiranSatyaRaj — COMMENTED** [review]
I agree with the changes required

File: content/en/cloud-computing.md (hunk 1 above)
Comment (iamNoah1) [inline content/en/cloud-computing.md:14]:
```suggestion 
```

File: content/en/cloud-computing.md [hunk 3]
```diff
@@ -10,7 +10,9 @@ tags: ["infrastructure", "", ""]
 Cloud computing is a model that offers compute resources like CPU, network, and disk capabilities on-demand over the internet. 
 Cloud computing gives users the ability to access and use computing power in a remote physical location. 
 Cloud providers like AWS, GCP, Azure, DigitalOcean, and others all offer third parties 
-the ability to rent access to compute resources in multiple geographic locations. 
+the ability to rent access to compute resources in multiple geographic locations.
+With a layer of [abstraction](/abstraction/) built on top of one another, this architecture is the foundation 
+for primary service models like [IaaS](/infrastructure-as-a-service/), [PaaS](/platform-as-a-service/), and [SaaS](/software-as-a-service/).
```
Comment (iamNoah1) [inline content/en/cloud-computing.md:15]:
```suggestion
```

**Review by iamNoah1 — CHANGES_REQUESTED** [review]
added actionable suggestions :)

**Review by iamNoah1 — APPROVED** [review]

**Comment by iamNoah1** [comment]
thanks @KiranSatyaRaj :)

**Review by CathPag — APPROVED** [review]
LGTM

**Pull request merged**